composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": "^7.2||^8.0",
-        "doctrine/annotations": "^1.0",
+        "doctrine/annotations": "^1.11",
```

Comment:
why do we need this specific version?

Reply:
Because any lower with "lowest" set of dependencies fails on PHP8. I've traced it to this though I did not find any particular change that causes it https://github.com/doctrine/annotations/compare/1.10.4...1.11.0. See failures here http://github.com/simPod/serializer/commit/20cf7008

Reply:
Ok, might have been some issue in my local installation. Bumping orm is enough.